test/Interop/Cxx/stdlib/use-std-span.swift

```diff
@@ -69,29 +69,29 @@ func takesSpanOfString(_ s: inout SpanOfString) {
   expectEqual(s[2], "abc")
 }
 
-func returnsConstSpanOfInt() -> ConstSpan {
+func returnsConstSpanOfInt() -> ConstSpanOfInt {
   let arr: [Int32] = [1, 2, 3]
```

Comment:
Some of these test functions contain UB. I plan to fix these tests in a separate PR.